pkg/services/store/service.go

```diff
@@ -86,3 +107,60 @@ func (s *standardStorageService) Read(ctx context.Context, user *models.SignedIn
 	// TODO: permission check!
 	return s.tree.GetFile(ctx, path)
 }
+
+func (s *standardStorageService) Upload(ctx context.Context, user *models.SignedInUser, form *multipart.Form) (*Response, error) {
+	response := Response{
+		path:       "upload",
+		statusCode: 200,
+		message:    "Uploaded successfully",
+		err:        false,
+	}
+	upload, _ := s.tree.getRoot("upload")
+	if upload == nil {
+		return &response, fmt.Errorf("upload feature is not enabled")
```

Comment:
This may be due to my limited understanding of go, but in this case are we still returning a 200 response in the event that upload is not enabled? Would it make more sense to return a `400` or maybe even a `404` status code in this case?

Reply:
oof yes you are right, this would return a 200. Will return a 404 instead.